xbmc/LangInfo.cpp

```diff
@@ -300,6 +302,7 @@ void CLangInfo::CRegion::SetGlobalLocale()
     strLocale = lcl.name();
     current_locale = current_locale.combine< std::collate<wchar_t> >(lcl);
     current_locale = current_locale.combine< std::ctype<wchar_t> >(lcl);
+    current_locale = current_locale.combine< std::time<wchar_t> >(lcl);
```

Comment:
I already tested ```std::time_get``` and ```std::time_put``` which are the right ones, but they didn't solve the problem I want to fix.